rbi/tools/sync-rdoc.rb

```diff
@@ -147,6 +147,11 @@ def initialize(file)
         walk_scope(scope, &blk)
         assert_clean!
       end
+    when :SCLASS
+      # There are a handful of places in RBIs where we have to use `class <<
+      # self` because both a class and it's singleton class need to register a
+      # type_member/type_template with the name Elem
+      puts 'skipping :SCLASS node, documentation will not be generated'
```

Comment:
cc @univerio if you have an idea for how hard it would be to handle `class << self` nodes in this script, I'd love your input. For example, this is the file the script choked on:

https://github.com/sorbet/sorbet/blob/e86efdfd1e9b1f3f55a6c917a27385c7d03de21e/rbi/stdlib/etc.rbi#L564-L571

Reply:
Oof, it's been a while since I've looked at this. Let me dig into this.

Reply:
Turns out it was really simple:

https://github.com/univerio/sorbet/commits/sync-rdoc-2-7

It's spewing a bunch of warnings due to some constants missing from ri though. 😕 Not sure what's going on there.

Reply:
Thanks!

I've merged this PR. Feel free to open a new one on latest master.